buildSrc/src/main/kotlin/convention.library.android.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("com.redmadrobot.android-library")
+    id("convention.publishing")
+    id("convention.detekt")
+}
+
+redmadrobot {
+    android.minSdk = 19
```

Let's make it 21, following trend in AndroidX libs
```suggestion
    android.minSdk = 21
```